geocoordinates/geocoordinates.php

```diff
@@ -26,34 +26,41 @@ function geocoordinates_uninstall()
 
 function geocoordinates_resolve_item(&$item)
 {
-	if((!$item["coord"]) || ($item["location"]))
+	if (!isset($item['coord']) || isset($item['location'])) {
 		return;
+	}
 
 	$key = Config::get("geocoordinates", "api_key");
-	if ($key == "")
+
+	if ($key == "") {
```

Comment:
`$foo == ""` can be replaced with `empty($foo)`, should I do so?

Reply:
Then the `if()` block below can be removed?

Reply:
Yes, provided you add the default to the Config::get call.

Reply:
Removed.